app/src/main/java/com/example/concordia_campus_guide/Helper/PathFinder.java

```diff
@@ -74,6 +83,10 @@ public boolean test(final WalkingPoint walkingPoint) {
         return optionalWalkingPoints.isPresent() ? optionalWalkingPoints.get() : null;
     }
 
+    /**
+     * This methods' purpose is to find a solution path from a point A to a point B. It will use a priorityQueue to get the best next point to visit at each moment. If the point was already visited, it will skip it.
```

Comment:
```suggestion
     * This method's purpose is to find a solution path from a point A to a point B. It will use a priorityQueue to get the best next point to visit at each moment. If the point was already visited, it will skip it.
```